alf/summary/summary_ops.py

```diff
@@ -85,24 +85,47 @@ def __exit__(self, type, value, traceback):
         _scope_stack.pop()
 
 
+_SUMMARY_DATA_BUFFER = {}
+
+
 def _summary_wrapper(summary_func):
     """Summary wrapper
 
     Wrapper summary function to reduce cost for data computation
     """
 
     @functools.wraps(summary_func)
-    def wrapper(name, data, step=None, **kwargs):
-        if should_record_summaries():
+    def wrapper(name, data, average_over_interval=False, step=None, **kwargs):
+        if average_over_interval:
```

Comment:
`average_over_interval` -> `average_over_summary_interval` to make it clearer about what the interval is?